### ISSUE TITLE
Deploy & Test Webservice inline script 

### ISSUE THREAD

**Issue by 105306002** [microsoft/MCW-ML-Ops#8]
inline script should be  

python $(System.DefaultWorkingDirectory)/_mlops-quickstart/devops-for-ai/aml_service/deploy.py --service_name $(service_name) --aks_name $(aks_name) --aks_region $(aks_region) --description $(description)

not  

python aml_service/deploy.py --service_name $(service_name) --aks_name $(aks_name) --aks_region $(aks_region) --description $(description)

and working Directory should be 
$(System.DefaultWorkingDirectory)/_mlops-quickstart/devops-for-ai

otherwise, it won`t deploy successed

**Comment by ciprianjichici** [comment]
We were not able to reproduce the issue. Can you please provide a more detailed description of the error you are getting during the **Deploy & Test Web Service** step of the release pipeline?

**Comment by 105306002** [comment]
it will say can not find aml_service/deploy.py  and can not open eval_info .json

**Comment by shirolkar** [comment]
Hi @105306002 @timahenning @DawnmarieDesJardins @ciprianjichici 

I tested this issue and was also not able to reproduce. It was sufficient to set the working directory without explicitly setting the same path again in the inline script. 
First, I did see this issue, but it was _user error_ since I used a different project name and copied the working directory path as: **$(System.DefaultWorkingDirectory)/_mlops-quickstart/devops-for-ai**. This path implies that the project name is: **mlops-quickstart**.
One way to confirm that you are setting the correct working directory path is to select the object browser and then select **devops-for-ai** as your working directory. See attached image.
Thanks. 
Jitendra 

<img width="1164" alt="Screen Shot 2019-11-01 at 10 15 09 AM" src="https://user-images.githubusercontent.com/18037946/68070093-fbc0e300-fd3f-11e9-81a3-f771fcbfada6.png">
